docs/section19/19-11.md

```diff
@@ -0,0 +1,41 @@
+# Section 19 - Rule 19-11         
+**Schema Version:** 0.0.22  
+**Mandatory Rule:** False    
+**Rule ID:** 19-11         
+**Rule Description:** For systems that serve computer rooms, if the  baseline system is HVAC System 11, it shall include an integrated fluid economizer meeting the requirements of Section 6.5.1.2 in the baseline building design.  
```

Comment:
I beleive in this case the chilled water plant has to be modeled with fluid economizer, so this would affect all systems 7 and 8 in the project. So neither of these would be modeled with air-side economizer then.

Reply:
Ignore, since no explicit exception in Appendix G. Outcome per phone call.